pkg/server/registry/apps/validator.go

```diff
@@ -319,6 +325,19 @@ func checkMemory(memory v1.Memory, workloads map[string]v1.Container, specMemDef
 	return validationErrors
 }
 
+func validateMemoryRunFlags(memory v1.Memory, workloads map[string]v1.Container) *field.Error {
```

Comment:
Should this logic just be in `v1/memory.go` file as apart of `v1.ValidateMemory`?

Reply:
I thought about that but it seemed like v1.validatememory handled more of the quantity logic/validation so it made sense to me to separate them out open to changing to other way if you or others have a strong opinion

Reply:
Only real difference is that this same logic won't happen on the controller unless it exists in `v1.ValidateMemory`. Right now validation happens on both the controller and the API server.

Reply:
ahh okay in that case I will move to v1.validatememory

Reply:
Discussed offline, but we'll go ahead and put this into an issue an track it as a follow-up.